### DISCUSSION TITLE
Getting error while running make command

### DISCUSSION THREAD

**Discussion by hardud** [idaholab/moose#27219] [Q&A Getting Started] [answered]
<img width="1440" alt="Screenshot 2024-03-28 at 5 13 53 PM" src="https://github.com/idaholab/moose/assets/144852214/5e1ed2cf-888b-4a6f-810a-25eab9d42323">

I have been trying to remove pytorch and install it again using pip. Then trying to configure and recompile moose (as suggested by @hugary1995  ), but still I am running into errors. I am using apple silicon. Please let me know if someone can fix the issue or has run into similar errors before. Thank you.

**Comment by GiudGiud** [comment]
Hello

Which set of instructions are you following for installing moose with torch?

Guillaume

**Reply by hardud** [reply]
For installing, I followed the instructions mentioned in slides during the Solid Mechanics workshop by Dr Hu.

**Reply by GiudGiud** [reply]
@grmnptr where are the official instructions for installing this?
this page doesnt seem super complete? https://mooseframework.inl.gov/modules/stochastic_tools/enable_pytorch.html

**Reply by hardud** [reply]
I have deleted all the moose files and uninstalled previous versions of moose. 
Now while installing moose again, my tests are failing. I did not have 
<img width="1176" alt="Screenshot 2024-03-31 at 4 08 17 PM" src="https://github.com/idaholab/moose/assets/144852214/8909de11-5f62-4a5d-ac4a-7f366e07c8f3">
any build issues.

**Reply by GiudGiud** [reply]
Hello

is this on a macOS machine?
I think we have an entry in the FAQ for this exact error and how to fix it 

Guillaume

**Reply by hardud** [reply]
Yes I am using macOS. While running the make command, I see 100s of these warnings. Are they fine or is it a trouble? If it is, can you help me encounter them? 
Thanks,
Harsh
<img width="1440" alt="Screenshot 2024-03-31 at 6 33 23 PM" src="https://github.com/idaholab/moose/assets/144852214/798d75fe-c495-4b9c-ba49-5eac535f2d8a">

**Reply by GiudGiud** [reply]
These are fine. Just the compiler outputting more warnings than it used to and we haven’t gotten to resolving them

**Reply by hardud** [reply]
<img width="1431" alt="Screenshot 2024-04-16 at 6 11 56 PM" src="https://github.com/idaholab/moose/assets/144852214/11304127-12c6-4c69-a47d-276a90ab1c25">

I am getting this information every time I am trying to run tutorials. Can you suggest how do I tackle them?

Thanks,
Harsh

**Reply by GiudGiud** [reply, marked as answer]
you forgot the -i

it should be 
```
......... / solid_mechanics-opt -i weak_plane_stress_elastic_jacobian.i
```

**Reply by hardud** [reply]
Oh, that was so silly of me. 

Thank you!
